packer_update_centos_base.json

```diff
@@ -0,0 +1,51 @@
+{
+  "variables" : {
+    "build_date" : "{{env `BUILD_DATE`}}"
+  },
+  "builders" : [
+    {
+      "type": "amazon-ebs",
+      "name" : "CentOS 6",
+      "region": "us-east-1",
+      "source_ami": "ami-b7c484a0",
+      "instance_type": "t2.micro",
+      "ssh_username": "centos",
+      "ssh_pty" : true,
+      "ami_name": "CentOS 6.x x86_64 - minimal with cloud-init - {{user `build_date`}}-0",
+      "associate_public_ip_address" : true,
+      "sriov_support" : true,
+      "ena_support" : true
+    },
+    {
+      "type": "amazon-ebs",
+      "name" : "CentOS 7",
+      "region": "us-east-1",
+      "source_ami": "ami-9dc3838a",
+      "instance_type": "t2.micro",
+      "ssh_username": "centos",
+      "ssh_pty" : true,
+      "ami_name": "CentOS 7.x x86_64 - minimal with cloud-init - {{user `build_date`}}-0",
+      "associate_public_ip_address" : true,
+      "sriov_support" : true,
+      "ena_support" : true
+    }
+  ],
+  "provisioners" : [
+    {
+      "type" : "shell",
+      "expect_disconnect" : true,
+      "inline" : [
+        "sudo yum -y upgrade",
+        "echo Update Complete.  Rebooting.",
+        "sudo reboot ; sleep 30"
```

Comment:
In case someone wonders, the sleep is because centos6 reboot returns before ssh shut down, so the next set of commands started running, and then the build failed when the instance actually rebooted.  The sleep keeps this set of shell commands from returning so quickly, so the instance reboot starts before the next set of shell provisioners starts.